src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -0,0 +1,410 @@
+    risks: float
+    overall: float
+    
+    def to_dict(self) -> Dict[str, float]:
+        """Convert to dictionary."""
+        return {
+            "novelty": self.novelty,
+            "impact": self.impact,
+            "cost": self.cost,
+            "feasibility": self.feasibility,
+            "risks": self.risks,
+            "overall": self.overall
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)